src/js/components/agency/overview/AgencyOverview.jsx

```diff
@@ -23,6 +30,44 @@ export default class AgencyOverview extends React.Component {
             logo = (<img src={this.props.agency.logo} alt={this.props.agency.name} />);
         }
 
+        // Move props to variables for readability
```

Comment:
Can we move a lot of this to `componentWillReceiveProps` and store the calculated values in state?